html/inc/boinc_db.inc

```diff
@@ -182,9 +182,16 @@ class BoincUser {
     }
     static function lookup_email_addr($email_addr) {
         $db = BoincDb::get();
-        $email_addr = BoincDb::escape_string($email_addr);
+        $email_addr = strtolower(BoincDb::escape_string($email_addr));
         return self::lookup("email_addr='$email_addr'");
     }
+    //This is to find email addresses that have changed in the past 7 days.
+    static function lookup_prev_email_addr($email_addr) {
+        $db = BoincDb::get();
```

Comment:
`$db` is unused.

Reply:
I'm going to make the change to remove it, also from a few other functions that call escape_string right after it, because within that function, it calls get().